src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,35 +56,40 @@ public String todayDate(DateTimePart datePart) {
      * Return Optional of a date built from these elements.
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
+        try {
+            return Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]));
+        } catch (DateTimeException | ArrayIndexOutOfBoundsException e) {
+            e.printStackTrace();
```

Comment:
You forget what Bohdan call "Bad practise"?)Print stack trace

Reply:
I disagree with you, what exactly is "bad practice"?